Sources/Sentry/SentryProfiler.mm

```diff
@@ -64,6 +64,36 @@
     return false;
 #    endif
 }
+
+// Parses a symbol that is returned from `backtrace_symbols()`, which encodes information
+// like the frame index, image name, function name, and offset in a single string.
+// e.g.
+// For the input:
+// 2   UIKitCore                           0x00000001850d97ac -[UIFieldEditor
+// _fullContentInsetsFromFonts] + 160 This function would return:
+// -[UIFieldEditor _fullContentInsetsFromFonts]
+//
+// If the format does not match the expected format, this returns the input string.
+NSString *
+parseFunctionName(const char *symbol)
```

Comment:
`m`: Can we add tests for this new method?

Reply:
done!